packages/server/src/modules/app/index.ts

```diff
@@ -56,3 +59,21 @@ application.start = () => {
   );
   socket.init(server);
 };
+
+if (process.env.LOCALHOST_CERTIFICATE_BASE_PATH) {
+  https
```

Comment:
This means server can run on both http and https in the same time, I'd avoid such behaviour for now.

Do following:
- Create `startHttpServer` function in app module that will do what's now done in start function above
- Create `startHttpsServer` function in app module that will have this logic here
- Create `shouldStartHttpsServer` function in app module that will check if env is configured for https

Then in `start` function above (in this file) run proper server.